neuraxle/base.py

```diff
@@ -282,7 +284,7 @@ def load_step(self, step: 'BaseStep', context: 'ExecutionContext') -> 'BaseStep'
 
         # we need to keep the current steps in memory because they have been deleted before saving...
         # the steps that have not been saved yet need to be in memory while loading a truncable steps...
-        if isinstance(step, TruncableSteps):
+        if isinstance(loaded_step, TruncableSteps) and hasattr(step, 'steps'):
```

Comment:
huge code smell. trying to find a way to clean this, but I have not found it yet....